modules/openapi-generator/src/main/resources/JavaSpring/api.mustache

```diff
@@ -7,7 +7,9 @@ package {{package}};
 
 {{#imports}}import {{import}};
 {{/imports}}
+{{#useDocumentationAnnotations}}
```

Comment:
so what is removing the io.swagger.annotations from the imports array? Otherwise it still gets added even if useDocumentationAnnotations is false. AND wouldn't it be better to handle this higher up, aka the JavaAbstractGenerator class so that anything using Java will remove the annotations from imports?